doc/project/rfc_process.md

```diff
@@ -22,17 +22,22 @@ This document outlines the basic structure of the RFC process, with the expectat
    * If requested, the TC could help to nominate a small group to shepherd the RFC.
 * If the RFC potentially contains information that could be certification-sensitive (guidance to be shared), send a note to security@opentitan.org first for feedback.
 * The RFC is shared publicly by filing a GitHub issue and tagging with the `RFC:Proposal` label.
-* Once the author is happy that the RFC is complete, they submit it to the Technical Committee for review by adding the Github label `TC Review`.
+* Once the author is happy that the RFC is complete, they submit it to the Technical Committee by converting the label to `RFC:TC Review`.
 * The Technical Committee will consider active RFCs in each meeting (those that have been marked ready for at least a week).
   If an RFC saw large changes in the week it has been "ready" the TC may postpone judgement in order to allow more comment.
   They will decide whether to:
    * **Accept**: Indicate they are happy for this proposal to be implemented.
      May be conditional on certain changes.
-   * **Reject**: Decided against the proposal at this time
+     The TC will change the label from `RFC:TC Review` to `RFC:Approved` in this outcome.
+   * **Reject**: Decided against the proposal at this time.
+     The label `RFC:TC Review` will be removed by the TC.
    * **Postpone**: The proposal seemed to have the right level of detail and form, but the project won't be making a decision on it at this time (e.g. refers to work that is scheduled too far in the future).
```

Comment:
I was wondering if this will always be the case. What if the TC just postpones it because there was not enough time in the meeting to review it. Would it be then in the responsibility of the submitter to change the label back to `RFC:TC Review` again? Or would this not be a "postpone". Or in other words: Since we are saying "The Technical Committee will consider active RFCs in each meeting" in line 26, does that mean the `RFC:TC Review` label of any TC can never survive a TC meeting?

Reply:
Could be clarified in the TC meeting. If TC does not actually review an RFC (e.g. because not enough time) it is not the "postponed" outcome but it just keeps the `RFC:TC Review` label. So no need to change wording here.